be/src/vec/core/block.h

```diff
@@ -75,6 +75,7 @@ class Block {
     IndexByName index_by_name;
     std::vector<bool> row_same_bit;
 
```

Comment:
Add some comment here to describe why need a _effective_col

Reply:
fixed, add `set_effective_col` comment